src/commands.rs

```diff
@@ -396,6 +396,9 @@ struct AbandonArgs {
     /// The revision(s) to abandon
     #[arg(default_value = "@")]
     revisions: Vec<RevisionArg>,
+    /// Do not print every abandoned commit on a separate line
+    #[arg(long, short)]
+    quiet: bool,
```

Comment:
I felt like the new output is too verbose and I was wondering if the short output should be the default and the verbose output should be behind a `--verbose`, but that flag is about verbose logging. Maybe we should rename it to `--debug`. I actually also expected `--quiet` to not show any output at all, but I guess "quiet" doesn't mean "silent" :) I don't feel strongly about any of that, so let's leave it as is. It's quite unusual that I abandon tens of commits, so I won't be surprised if I end up liking this new default output.

Reply:
I agree with your point about `--quiet`, I consider that to be a minor bug that needs to be fixed eventually.

The Discord discussion raised a related issue for me: `--verbose` for informational commands or errors makes a lot of sense. However, having `--verbose` print additional output for mutating commands can be annoying since when you realize you needed the extra output, it's already too late.

I'm currently leaning towards leaving this as is, and seeing if it becomes annoying to have it be this verbose.

Reply:
Following our discussion on Discord, I feel like we should rename the flag from `--quiet` to `--summary` or something like that, so we make the flag names more descriptive. Then we might later add `--quiet` as a global flag that removes all non-error, non-warning output.

Reply:
Sounds good.